README.md

```diff
@@ -63,8 +63,7 @@ usage: glasbey.py [-h] [--base-palette BASE_PALETTE] [--no-black] [--view]
 
     If having black (and colors close to black) is undesired, then `--no-black`
     option may be used to prevent the algorithm from inserting such colors into
-    the palette. In addition to that, the range of colors considered for
-    inclusion in the palette can be limited by lightness, chroma, or hue.
+    the palette.
```

Comment:
Some documentation got lost, could you please add it back? Also positional arguments below.